TradeItIosTicketSDK2/TradeItLinkedBrokerManager.swift

```diff
@@ -26,7 +26,7 @@ import PromiseKit
         let tradeItSession = tradeItSessionProvider.provide(connector: self.tradeItConnector)
         let linkedBroker = TradeItLinkedBroker(session: tradeItSession, linkedLogin: linkedLogin)
         
-        //we need to provide an error when we load from the keychain in order to the authenticate all method to handle this linkBroker
+        // TODO: Refactor the error code so we don't need to provide an error when we load from the keychain to indicate unauthenticated
```

Comment:
I actually don't think this needs to be refactored. It's kind of just like a dirty flag.